src/components/Content/ContentMining.vue

```diff
@@ -28,7 +28,7 @@
               :src="require('@/assets/art/mining/icon.png')"
             />
             <b>Mine</b> some
-            <img class="mx--2" :src="require('@/assets/art/mining/SheetIron.png')" />
+            <img class="mx--2" :src="require('@/assets/art/mining/SheetGlass.png')" />

```

Comment:
If you're updating one of these **Ore** icons, you should update them all for consistency.

ContentMining.vue line 20
ContentMining.vue line 50
ContentMining.vue line 81
ContentFabrication.vue line 37
ContentFabrication.vue line 47

Reply:
I deliberately left these in as metals are the main result of mining activities. Having iron as the base makes more thematic sense, this is also why I added the bit about beginners pounding sand. "Yes, you are here for metal. Nah, pound sand instead noob".

Reply:
That's fair. I'd still request that this line, and line 20 be the same though as there's sort of a connection between the two
![image](https://user-images.githubusercontent.com/16600313/87176998-eb86dd80-c2a8-11ea-8fb8-fce2fd08aafd.png)
I'd be fine with them either both being glass, or both being iron

Sorry if this sounds a bit nitpicky, it's just that it's the first thing new players see so it needs to be as clear as possible.

Reply:
understandable, I'll switch it to metal.